pkg/render/monitor_test.go

```diff
@@ -71,12 +73,83 @@ var _ = Describe("monitor rendering tests", func() {
 
 			if alertmanagerObj, ok := obj.(*monitoringv1.Alertmanager); ok {
 				com := components.ComponentPrometheusAlertmanager
-				expectedImage := fmt.Sprintf("%s%s:%s", components.PrometheusRegistry, com.Image, com.Version)
```

Comment:
Rather than iterating over all objects and trying to cast them, we have some logic to help you find the object you are looking for. This is how we do it in other tests, I would suggest you use that behaviour as well. For instance when you want to find a certain deployment from the rendered objects:
```
//dex_test.go:150
d := rtest.GetResource(resources, render.DexObjectName, render.DexNamespace, "apps", "v1", "Deployment").(*appsv1.Deployment)
Expect(...).To(...)
```

Reply:
As suggested, moved to a separate test case. Thank you!